pkg/apis/stork/v1alpha1/applicationclone.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	//ApplicationCloneResourceName is the name for the application clone resource
+	ApplicationCloneResourceName = "applicationclone"
+	//ApplicationCloneResourcePlural is the name in plural for the application clone resources
+	ApplicationCloneResourcePlural = "applicationclones"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+//ApplicationClone represents the cloning of application in different namespaces
+type ApplicationClone struct {
```

Comment:
@disrani-px , @sharma-tapas ApplicationClone sounds little misleading. It feels like it is a clone of an application. While this object is basically carrying out the migration between namespaces. Could it be called NamespaceMigration?

Reply:
It will be cloning an application between namespaces, that's what users will want to do from their perspective. 
I want a consistent naming for the other new CRDs we are adding for application operations (ApplicationBackup and ApplicationRestore)

Reply:
Keeping it as ApplicationClone to be inline with what @disrani-px said, we can change it later on if it becomes confusing.